Web/starter_project_barbies/types/team/TeamMemberCardProps.ts

```diff
@@ -0,0 +1,9 @@
+import { SocialMediaLink } from './SocialMediaLink'
+
+export type TeamMemberCardProps = {
```

Comment:
i know @Adona12 told you to do this but i prefer keeping prop definitions right above the component definition

Reply:
I would also still like you to take in just one TeamMember object that is defined here instead 
so basically I think this definition is more for a TeamMember object and the page props definition will take it as an object (which would correspond better with what you would find in the database)

Reply:
> i know @Adona12 told you to do this but i prefer keeping prop definitions right above the component definition

So, should I change it back?